test/mocks/Battle.spec.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+const assert = require('../assert');
+const Battle = require('./Battle');
+
+describe('Battle', () => {
+	it('should reset the PRNG to original when I call resetRandomNumberGenerator', () => {
```

Comment:
Good. By the way, [Arrow functions are discouraged in Mocha.js](http://mochajs.org/#arrow-functions)